### PR TITLE
Rus/models

### PR DESCRIPTION
**Pull request by RuslanProgrammer**
Login screen

### CODE REVIEW

File: lib/screens/AuthenticationScreen.dart [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            ),
+            SizedBox(
+              height: 20,
+            ),
+            Padding(
+              padding: EdgeInsets.only(left: 20, right: 20, bottom: 5),
+              child: Container(
+                height: 50,
+                width: MediaQuery.of(context).size.width,
+                child: _button(label, foo),
```
Comment (chummy-ch) [inline lib/screens/AuthenticationScreen.dart]: You should not use constant values for the widget scalse, coz it isn`t flexible

File: lib/screens/AuthenticationScreen.dart [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      }
+      if (!passwordValidator.validate(_password)) {
+        showToast("Please check password\nThen repeat again");
+        return;
+      }
+
+      MyUser.User user =
+          await _authService.signInWithEmailAndPassword(_email, _password);
+      if (user == null) {
+        showToast("Please check email and password\nThen repeat again");
```
Comment (chummy-ch) [inline lib/screens/AuthenticationScreen.dart]: We need to use string resources for better interactions and to give an opportunity to translate our app

File: lib/utils/AuthService.dart [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:water_overflow/models/User.dart' as myUser;
+
+class AuthService {
+  final FirebaseAuth _fAuth = FirebaseAuth.instance;
```
Comment (chummy-ch) [inline lib/utils/AuthService.dart:6]: This value must be private if we don't use it somewhere outside of our class

**Review by chummy-ch — CHANGES_REQUESTED** [review]

File: lib/utils/AuthService.dart (hunk 3 above)
Comment (RuslanProgrammer) [reply lib/utils/AuthService.dart:6]: > This value

class or variable?

Comment (chummy-ch) [reply lib/utils/AuthService.dart:6]: The variable that is using only inside current class

**Review by chummy-ch — CHANGES_REQUESTED** [review]
You should pull developer branch

**Review by chummy-ch — APPROVED** [review]

**Pull request merged**